apk-generator/v2/app/static/js/utils.js

```diff
@@ -2,13 +2,13 @@ jQuery.fn.extend({
         /**
          * An extended jQuery method to call a callback on an input change
          */
-        valueChange: function (callback) {
+        valueChange(callback) {
```

Comment:
This is es6 syntax. Not supported in browsers without a transpiler like babel. Revert this back please.